tests/fbe_factory.pm

```diff
@@ -14,6 +14,14 @@ sub run {
 
     # wait for and dismiss factory dialog
     assert_and_click('fbe_factory_dialog', 'left', 10);
+    assert_screen('fbe_welcome', 10);
+
+    # test that VT-switching to a console before the FBE is complete gives a
+    # login prompt, not "Hold until boot process finishes up".
+    # https://phabricator.endlessm.com/T21181
+    send_key("ctrl-alt-f4");
+    assert_screen('text_console_login', 10);
+    send_key("ctrl-alt-f1");
```

Comment:
I’m on the fence about whether this should be in `fbe_factory` or in a new `fbe_*` test of its own. Probably fine here for now.

Reply:
I was planning to rename the test to something suggesting "test the fbe itself, rather than just going through it as fast as possible"